test/ci/backend/test_lava.py

```diff
@@ -33,6 +35,8 @@
     {'duration': '',
      'job': '1234',
      'level': 'None',
+     'log_end_line': '4',
+     'log_start_line': '5',
```

Comment:
`log_start_line` is greater than `log_end_line`

Reply:
I know. I did that on purpose.

Reply:
Oh, nvm then :)